AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -118,13 +118,18 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
     with Tracer() as error_tracer:
         # Set constants before starting test steps.
         standard_pbr_material_type = os.path.join(atom_tools_utils.MATERIAL_TYPES_PATH, "StandardPBR.materialtype")
-        test_data_path = os.path.join(atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases")
-        test_material_1 = "001_DefaultWhite.material"
-        test_material_2 = "002_BaseColorLerp.material"
-        test_material_3 = "003_MetalMatte.material"
-        test_material_4 = "004_MetalMap.material"
-        test_material_5 = "005_RoughnessMap.material"
-        test_material_6 = "006_SpecularF0Map.material"
+        test_material_1 = os.path.join(
+            atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases", "001_DefaultWhite.material")
+        test_material_2 = os.path.join(
+            atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases", "002_BaseColorLerp.material")
+        test_material_3 = os.path.join(
+            atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases", "003_MetalMatte.material")
+        test_material_4 = os.path.join(
+            atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases", "004_MetalMap.material")
+        test_material_5 = os.path.join(
+            atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases", "005_RoughnessMap.material")
+        test_material_6 = os.path.join(
+            atom_tools_utils.TEST_DATA_MATERIALS_PATH, "StandardPbrTestCases", "006_SpecularF0Map.material")
```

Comment:
NIT: A local helper function will help make this more readable.

Reply:
I'm doing something similar to this in another py file. this will shorten the number of imports in the test itself and make it even more readable.

Reply:
I utilize them as constants for other values throughout the test steps and these materials are only specific to this test script, going to leave it as is for the time being